syntax = "proto3";
package com.panelsw.ca;
option optimize_for = LITE_RUNTIME;

message UnzipDetails{

	enum OverwriteMode {
		never = 0;
		always = 1;
		unmodified = 2;
	}

    bytes zipFile = 1;
    bytes targetFolder = 2;
	OverwriteMode overwriteMode = 3;
}
